.taskcluster.yml

```diff
@@ -2,341 +2,346 @@
-                                            - index.mobile.v2.${project}.branch.${head_ref}.revision.${head_sha}.taskgraph.cron.${ownTaskId}
-                        scopes:
-                            $if: 'tasks_for == "github-push"'
-                            then:
-                                - 'assume:repo:github.com/${repoFullName}:branch:${short_head_ref}'
-                            else:
-                                $if: 'tasks_for == "github-pull-request"'
+                                            kind: cron-task
+                            routes:
+                                $flattenDeep:
+                                    - checks
+                                    - $if: 'level == "3" || repoUrl == "https://github.com/mozilla-releng/staging-firefox-android"'
+                                      then:
+                                          - tc-treeherder.v2.${project}.${head_sha}
+                                          - $if: 'tasks_for == "github-push"'
+                                            then:
+                                                - index.mobile.v2.${project}.branch.${short_head_ref}.latest.taskgraph.decision
+                                                - index.mobile.v2.${project}.branch.${short_head_ref}.revision.${head_sha}.taskgraph.decision
+                                                - index.mobile.v2.${project}.revision.${head_sha}.taskgraph.decision
+                                          - $if: 'tasks_for == "cron"'
+                                            then:
+                                                # cron context provides ${head_ref} as a short one
+                                                - index.mobile.v2.${project}.branch.${head_ref}.latest.taskgraph.decision-${cron.job_name}
+                                                - index.mobile.v2.${project}.branch.${head_ref}.revision.${head_sha}.taskgraph.decision-${cron.job_name}
+                                                - index.mobile.v2.${project}.branch.${head_ref}.revision.${head_sha}.taskgraph.cron.${ownTaskId}
+                            scopes:
+                                $if: 'tasks_for == "github-push"'
                                 then:
-                                    - 'assume:repo:github.com/${repoFullName}:pull-request'
+                                    - 'assume:repo:github.com/${repoFullName}:branch:${short_head_ref}'
                                 else:
-                                    $if: 'tasks_for == "action"'
+                                    $if: 'isPullRequest'
                                     then:
-                                        # when all actions are hooks, we can calculate this directly rather than using a variable
-                                        - '${action.repo_scope}'
+                                        - 'assume:repo:github.com/${repoFullName}:${tasks_for[7:]}'
```

Comment:
I changed this line to support both `github-pull-request` and `github-pull-request-untrusted`, per https://github.com/taskcluster/taskgraph/blob/3481bb676219541dedbd9ef1efd27e0e0875e1eb/.taskcluster.yml#L222